src/osd/SnapMapper.cc

```diff
@@ -135,12 +140,21 @@ pair<snapid_t, hobject_t> SnapMapper::from_raw(
   return make_pair(map.snap, map.hoid);
 }
 
+std::pair<snapid_t, hobject_t> SnapMapper::from_raw(ceph::buffer::list& image)
```

Comment:
moved that '&'. Maintained the ceph:buffer:list, as this is the most common in the existing file.
Also - made that interface 'const'.

Reply:
This appears to be the only usage of ceph::buffer::list in this file, every other instance is bufferlist.  SnapMapper.h does, however, mostly use ceph::buffer::list.  ceph::buffer::list seems to outnumber bufferlist in osd/, perhaps it would be worth adding a commit to convert SnapMapper.cc?

Reply:
added a commit to that end